<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BarMenu</class>
 <widget class="QDialog" name="BarMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/StartWindow/Design/Фон/BarMenu.png);</string>
  </property>
  <widget class="QPushButton" name="pushButton_Exit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(207, 207, 207);</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Juice">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;Cascadia Code&quot;;
color: rgb(207, 207, 207)</string>
   </property>
   <property name="text">
    <string>Buy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Tea">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;Cascadia Code&quot;;
color: rgb(207, 207, 207)</string>
   </property>
   <property name="text">
    <string>Buy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cola">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;Cascadia Code&quot;;
color: rgb(207, 207, 207)</string>
   </property>
   <property name="text">
    <string>Buy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_PinkGirl">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;Cascadia Code&quot;;
color: rgb(207, 207, 207)</string>
   </property>
   <property name="text">
    <string>Buy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ShugarDaddy">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;Cascadia Code&quot;;
color: rgb(207, 207, 207)</string>
   </property>
   <property name="text">
    <string>Buy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Vodka">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;Cascadia Code&quot;;
color: rgb(207, 207, 207)</string>
   </property>
   <property name="text">
    <string>Buy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Whiskey">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;Cascadia Code&quot;;
color: rgb(207, 207, 207)</string>
   </property>
   <property name="text">
    <string>Buy</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
